### ISSUE TITLE
Q: Provider auditing

### ISSUE THREAD

**Issue by avolon42x** [HandyOSS/HandyHost#44]
Going down the rabbit hole a bit further. Here (https://docs.akash.network/operations/akash-audited-attributes) it says that providers should be audited in order to become "Akash" network providers. How can I do this with HH? Do I have to (makes sense so that you have more exposure on the network for serious hosting, right?)?

curious and learning.. thank you.

**Comment by avolon42x** [comment]
Do I understand correctly that by using HH for Akash I can't become an "official" host in terms of being singed? 
This would mean that a lot of bids like the ones with the following requirements cannot be met
host: Akash;
signed_by.any_of: akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63 (-> singed by a validator)

This would seriously limit handyhost..is that true? Thanks for clearing that up.

**Comment by alexsmith540** [comment]
I have tried to go down the rabbit hole of seeing how to get the official Akash signed_by as well. Basically some clients want specific people they audit to be their hosts and the single entity akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63 has all the power over signing providers. Using HH doesnt limit you from getting signed at all, it seems like the only limiting factor to getting signed is a formal process to get your provider signed.

**Comment by avolon42x** [comment]
Yeah - I get that. Can you make the provider attributes accessible in a way, so that necessary amendments could be made? From what I was told this would be a requirement. I think I found where to change it but of course with the next release it would be overwritten. I can make an improvement out this ticket?

**Comment by alexsmith540** [comment]
If you need additional fields added to the provider.yaml in order to get audited please let me know and I'll add them to the UI which populates provider.yaml. Ultimately there is already a UI to populate provider.yaml attributes already, it's in the configuration panel (anything listed under the Akash Provider Details area). The 3 attributes we currently are able to post in a provider.yaml are host (your URL), the region (ex: westcoast), and a host moniker (ex: my-host-moniker). Which ultimately populate a provider.yaml which is sent to akash, along with your provider's akash wallet address, when you register your provider. 
Example of what provider.yaml looks like currently:
```
host: https://my.provider.domain:8443
attributes:
  - key: region
    value: westcoast
  - key: host
    value: my-host-moniker
```

**Comment by avolon42x** [comment]
Thank you @alexsmith540 I was told that "tier" would be another attribute that needs to be set. I manually updated the provider.yaml will try to see if they sign my signature like that. 🤞 I'll let you know.

**Comment by alexsmith540** [comment]
Fixed in v0.5.2. 
Added tier, email, organization fields. Also if you rebuild your cluster in the Configuration UI, it will add gvisor.
